Makefile

```
@@ -246,6 +246,33 @@ restart-backend-utia:
 	@echo "Test connection: curl http://localhost:3001/api/test-email/test-connection"
 
 # Run unit tests with UI in Docker
+test-py:
```

checkmake (0.3.2)

[warning] Target body for "test-py" exceeds allowed length of 5 lines (6).

(maxbodylength)